frontend/src/reducer/notificationList.js

```diff
@@ -22,27 +28,49 @@ export default function notificationList (state = defaultNotificationList, actio
           type: no.event_type,
           workspace: no.fields.workspace ? no.fields.workspace : null
         }))
-        .reverse()
-      return uniqBy(notificationList, 'id')
+      return { ...state, list: uniqBy(notificationList, 'id') }
+    }
+
+    case `${APPEND}/${NOTIFICATION_LIST}`: {
+      const notificationList = action.notificationList
+        .map(no => ({
+          author: no.fields.author.public_name,
+          content: no.fields.content ? no.fields.content : null,
+          created: no.created,
+          id: no.event_id,
+          read: no.read,
+          type: no.event_type,
+          workspace: no.fields.workspace ? no.fields.workspace : null
+        }))
```

Comment:
this looks like it can be in a serializer
especially since it is repeated in several reducers

Reply:
Changed